### DISCUSSION TITLE
"add `--enable-features=DirectSockets` command line arguments." help

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#401] [Q&A] [answered]
"add `--enable-features=DirectSockets` command line arguments."
can I do this on a stable channel chromebook? and if so, how? I want to try the raw TPC backend, but I can't figgure out how to do this step.

**Comment by yume-chan** [comment, marked as answer]
See https://www.chromium.org/developers/how-tos/run-chromium-with-flags/#chrome-os
